bitbucket/data_plugin_test.go

```diff
@@ -7,6 +7,7 @@ import (
 )
 
 func TestAccBitbucketDataPlugin_notifyer(t *testing.T) {
+	t.Skip("Skipping testing in CI environment")
```

Comment:
These [pass ok in the CI environment](https://travis-ci.org/github/gavinbunney/terraform-provider-bitbucketserver/builds/615187581), is there a reason to skip all those tests?

Reply:
Hi @gavinbunney, we can remove this change to skip the tests, but FYI, the tests are really flaky as seen in this recent [build](https://travis-ci.org/github/gavinbunney/terraform-provider-bitbucketserver/builds/671483951) as well.

Reply:
I only observed `TestAccBitbucketPluginConfig` to be flaky while you are skipping 3 different tests here. Are you sure all of them are flaky? 

I personally dont believe in skipping the tests. We should rather debug this and make them stable.

Reply:
We are running all of the tests on our CI system as well and this test fails sometimes due to the plugin not being loaded and then the destruction of the resource fails with an API Error 500. This is likely due to the bitbucket status page replying with 200 response codes which the tests rely on to confirm that the Bitbucket server is up before the server is fully available. As a result the plugin is not able to load.
If the tests are run after a longer time (giving Bitbucket time to fully start up) then this test and the other plugin tests will pass consistently.
We can add these tests back for now and figure out how to resolve these failures later.

Reply:
@kamahl19 that is our preference as well but we don't always have enough time to both fix existing problems and implement new functionality.

Reply:
We've included these tests again